Makefile

```diff
@@ -41,10 +41,12 @@ AUTHOR ?= $(shell git config user.email || echo $$USER)
 ENABLE_METRICS ?= true
 BUILD_FLAGS ?= $(shell echo "-ldflags='\
 	-X github.com/status-im/status-go/params.Version=$(RELEASE_TAG:v%=%) \
+	-X github.com/status-im/status-go/params.IPFSGatewayURL=$(IPFSGatewayURL) \
```

Comment:
Call it `IPFS_GATEWAY_URL` please. And also you can set a default in the `Makefile` using the same `?=` lazy assignment so it can be overridden by environment variables when calling `make.`

Reply:
Good idea. Updated